emcc.py

```diff
@@ -2771,14 +2771,19 @@ def modularize():
       # after document.currentScript is gone, so we save it.
       # (when MODULARIZE_INSTANCE, an instance is created
       # immediately anyhow, like in non-modularize mode)
+      if shared.Settings.EXPORT_ES6:
+        script_url = "import.meta.url"
```

Comment:
this can use a comment - i'm not sure what it means?

Reply:
It's basically the same as `document.currentScript.src`, but for modules. `document.currentScript.src` is not available to modules (see: https://html.spec.whatwg.org/multipage/dom.html#dom-document-currentscript), so `import.meta.url` (or some other way) has to be used instead.

Basically the result is the same, just one works for modules and the other doesn't.
Do you still think this needs a comment?

Reply:
Thanks! Yeah, no need for a comment then.